### PR TITLE
[CLOSES #415] Webpack alias for src/ and tests/

### LINKED ISSUE #415: Add webpack aliases to support imports of path "src/*" and "tests/*"

### PR DESCRIPTION
**Pull request by aashnisshah**

### CODE REVIEW

**Comment by adrw** [comment]
CI is probably failing at this point because the package.json for tabs is being overwritten when `miskweb prebuild` is run by the currently published version of CLI. Fix is to publish an alpha (ie. `0.1.17-5`) that CI passes on. https://github.com/cashapp/misk-web/blob/b3d8be5f7e1b2ec1e1f64461d6cd531275a080f8/.circleci/config.yml#L21

**Comment by adrw** [comment]
2 AI:
- PR needs a rebase
- Please add a commit updating the `changelog.md` with any changes / migration steps for breaking changes in `0.1.17-4`

File: examples/tabs/skunkworks/NoDucksTabContainer.tsx [hunk 1]
```diff
@@ -1,20 +1,20 @@
-import { get, initialResponseState, OfflineComponent } from "@misk/core"
-import * as React from "react"
-import { SampleTableComponent } from "../components"
+import { get, initialResponseState, OfflineComponent } from "@misk/core";
+import * as React from "react";
+import { SampleTableComponent } from "src/components";
```
Comment (adrw) [inline examples/tabs/skunkworks/NoDucksTabContainer.tsx]: Nice 👍

File: examples/tabs/skunkworks/NoDucksTabContainer.tsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
-import { get, initialResponseState, OfflineComponent } from "@misk/core"
-import * as React from "react"
-import { SampleTableComponent } from "../components"
+import { get, initialResponseState, OfflineComponent } from "@misk/core";
+import * as React from "react";
```
Comment (adrw) [inline examples/tabs/skunkworks/NoDucksTabContainer.tsx]: These semi-colons everywhere look like Prettier didn't run. Try `miskweb lint -e` from the root directory to re-apply linting everywhere.

Comment (aashnisshah) [reply examples/tabs/skunkworks/NoDucksTabContainer.tsx]: Spoke offline with @adrw and realized the linting didn't work in `skunkworks` specifically. AI is to delete `skunkworks` in a future PR.

**Pull request merged**